wtxmgr/db.go

```diff
@@ -1215,18 +1215,39 @@ func (it *unminedCreditIterator) reposition(txHash *chainhash.Hash, index uint32
 //   [0:32]   Transaction hash (32 bytes)
 
 func putRawUnminedInput(ns walletdb.ReadWriteBucket, k, v []byte) error {
-	err := ns.NestedReadWriteBucket(bucketUnminedInputs).Put(k, v)
+	spendTxHashes := ns.NestedReadBucket(bucketUnminedInputs).Get(k)
+	spendTxHashes = append(spendTxHashes, v...)
```

Comment:
Also I _think_ we might actually need to copy these bytes to do an operation like this safely within `bolt`.

Reply:
I don't mind taking another approach, but the tests exercise this behavior and it seems to work. We could do something like below, but the current approach is much simpler:
```
if len(spendTxHashes)/32 == 0 {
    spendTxHashes = v
} else {
    // We allocate a new slice big enough to store the previous
    // entries and the new one.
    newSlice := make([]byte, len(spendTxHashes)+32)
    copy(newSlice, spendTxHashes)
    copy(newSlice[len(newSlice)-32:], v)
    spendTxHashes = newSlice
}
```